site/content/integrate/apps/authentication/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Authentication"
+heading: "Authentication"
+description: "TODO"
+weight: 200
```

Comment:
Maybe this page should be placed after `API` as `API` is the better stating point for new devs